app/assets/javascripts/application.js

```diff
@@ -20,9 +20,7 @@
 var ready;
 ready = function() {
   $(document).foundation();
-  $('.fdatepicker').fdatepicker({
-      format: "dd/mm/yyyy"
-  });
+  DependentDatePickersInit();
```

Comment:
Missing 'new' prefix when invoking a constructor.<br>'DependentDatePickersInit' is not defined.